src/components/header.js

```diff
@@ -1,8 +1,8 @@
 import '../styles/header.css';
 import logo from '../assets/logo/logo.svg';
 
-const Header = ({ userInfo, isSticky = false }) => {
-  const profileImage = userInfo?.profileImageSource;
+const Header = ({ userInfo, isSticky }) => {
+  const profileImage = userInfo?.profileImageSource ? userInfo?.profileImageSource : userInfo?.image_source;

```

Comment:
P3: 
불필요한 삼항연산자인것 같아요.

```suggestion
  const profileImage = userInfo?.profileImageSource ?? userInfo?.image_source;
```